app.js

```diff
@@ -80,8 +85,45 @@ const addBookHandler = () => {
          return;
     }
     book.addBook(bookTitle.value, bookAuthor.value);
+    bookTitle.value = "";
+    bookAuthor.value = "";
+    addBtn.parentElement.classList.add('hidden');
+    bookListUi.parentElement.classList.remove('hidden');
+    contactSection.classList.add('hidden');
     book.renderBook();
 }
 
 addBtn.addEventListener('click', addBookHandler);
+
+listNav.addEventListener('click', () => {
+  bookListUi.parentElement.classList.remove('hidden');
+  addBtn.parentElement.classList.add('hidden');
+  contactSection.classList.add('hidden');
+});
+
+addBookNav.addEventListener('click', () => {
+  addBtn.parentElement.classList.remove('hidden');
+  bookListUi.parentElement.classList.add('hidden');
+  contactSection.classList.add('hidden');
+});
+
+contactNav.addEventListener('click', () => {
+  contactSection.classList.remove('hidden');
+  addBtn.parentElement.classList.add('hidden');
+  bookListUi.parentElement.classList.add('hidden');
+});
+
+function getTime() {
+  const dateDiv = document.querySelector(".date");
+  dateDiv.textContent = new Date().toLocaleString('en-US', {
+    month: 'long',
+    day: 'numeric',
+    year: 'numeric',
+    hour: 'numeric',
+    minute: 'numeric',
+    second: 'numeric',
+  });
+}
+getTime();
```

Comment:
- [ ] Here you have to use [luxon](https://moment.github.io/luxon/#/install) library to display the current date as p-er the project requirement. The task here is to use a new tool effectively with little externally curated support. Exactly as you will do in your future job. Please use it.